src/lib/structures/errors.ml

```diff
@@ -26,8 +26,6 @@
 (*                                                                            *)
 (******************************************************************************)
 
-open Format
```

Comment:
I'm not sure removing this open is relevant as you open it in almost every other function.